.nycrc.json

```diff
@@ -0,0 +1,14 @@
+{
+  "all": true,
+  "include": [
+    "bin/*",
```

Comment:
Don't think I quite got this right, but I also don't think it was working before either